sigmf-spec.md

```diff
@@ -425,6 +428,11 @@ fields may be relative to baseband. It is required that both `freq_lower_edge`
 and `freq_upper_edge` be provided, or neither; the use of just one field is not
 allowed.
 
+The `label` field may be used for any purpose, but it should be limited to fewer
+than 20 charachters as a common use is a short form indicator for GUIs. As such,
+any user interface making use of this field should be capable of displaying up
+to 20 characters.
```

Comment:
I think we should make this "recommended" rather than a requirement, and should use RFC 2119 (per the [Conventions](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#conventions-used-in-this-document)). Suggestion:

```
The `label` field may be used for any purpose. It is RECOMMENDED that the label
be fewer than 20 characters to make it easily usable by GUIs.
```

Reply:
Agree this is more clear than `should` - fixed.